### PR TITLE
rework the home page to include recent builds

### PR DESCRIPTION
**Pull request by tardyp**
This new home page query the 20 last builds, and group them per builder

image

### CODE REVIEW

File: www/base/src/app/home/home.controller.coffee [hunk 1]
```diff
@@ -14,3 +14,20 @@ class Home extends Controller
 
         data = dataService.open($scope)
         $scope.builds_running = data.getBuilds(order: '-started_at', complete: false).getArray()
+        $scope.recent_builds = data.getBuilds(order: '-buildid', complete: true, limit:20).getArray()
+        $scope.builders = data.getBuilders().getArray()
+        buildersById = {}
+        $scope.hasBuilds = (b) -> b.builds.length > 0
+        update_builds = ->
```
Comment (tothandras) [inline www/base/src/app/home/home.controller.coffee]: Why do you prefer snake_case over camelCase lately? :)

**Comment by tardyp** [comment]
fixed the styling..

**Comment by sa2ajj** [comment]
Anyone to review? :)  (I'll merge it tomorrow roughly at the same time if noone volunteers :smile:)

**Comment by sa2ajj** [comment]
Sorry, it took me a bit longer :/

**Pull request merged**